lib/plugins/features/movies/index.js

```diff
@@ -1,7 +1,8 @@
 'use strict';
 
-const Controller     = require('./controller');
-const MovieValidator = require('../../../validators/movie');
+const Controller         = require('./controller');
+const MovieValidator     = require('../../../validators/movie');
+const MoviesGetValidator = require('../../../validators/movies_get');
```

Comment:
Usually for validators that all pertain to a certain resource, we create a [directory](https://github.com/lob/lob-api/tree/62d5671deeef5d74fffda6658088656f6a822221/src/validators/postcards) for them. So in this case, you'd have `movies/create` and `movies/list`.